.cloudtest/aws.yaml

```diff
@@ -10,7 +10,7 @@ providers:
     env:
       - CLUSTER_RULES_PREFIX=aws
       - AWS_CLUSTER_NAME=$(cluster-name)-$(date)-${CIRCLE_BUILD_NUM}-$(rands10)
-      - CONTAINER_REPO=networkservicemesh
+      - CONTAINER_REPO=networkservicemeshci
```

Comment:
I think better to move CONTAINER_REPO into env-check, to just ensure it is passed.

Reply:
OK will try and see how this works.